modules/node-shared/src/main/scala/org/tessellation/node/shared/domain/seedlist/SeedlistEntry.scala

```diff
@@ -0,0 +1,90 @@
+package org.tessellation.node.shared.domain.seedlist
+
+import cats.Order
+import cats.syntax.all._
+
+import org.tessellation.node.shared.domain.seedlist.SeedlistEntry.Alias
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust._
+
+import com.comcast.ip4s.IpAddress
+import derevo.cats.eqv
+import derevo.derive
+import eu.timepit.refined.auto._
+import fs2.data.csv._
+import fs2.data.csv.generic.semiauto.deriveRowDecoder
+import io.estatico.newtype.macros.newtype
+
+case class SeedlistEntry(
+  peerId: PeerId,
+  ipAddress: Option[IpAddress],
+  alias: Option[Alias],
+  bias: Option[TrustValueRefined],
+  ordinalTimeline: Option[HistoricalSnapshotOrdinalEntries]
+)
+
+object SeedlistEntry {
+
+  @derive(eqv)
+  @newtype
+  case class Alias(value: String)
+
+  object Alias {
+
+    implicit val optionalAliasCellDecoder: CellDecoder[Option[Alias]] =
+      CellDecoder.stringDecoder.map { s =>
+        Option.when(s.trim.nonEmpty)(Alias(s.trim))
+      }
+
+  }
+
+  implicit val maybeIpAddressCellDecoder: CellDecoder[Option[IpAddress]] =
+    CellDecoder.stringDecoder.emap { value =>
+      if (value.trim.nonEmpty)
+        IpAddress
+          .fromString(value)
```

Comment:
1. We're modifying the value, checking condition but if we keep it we're using un-modified value.
2. Are you sure you need this check? `IpAddress.fromString` already returns `None` on a blank string.

Reply:
I agree with point 1. Point 2 is probably related with what I have asked: https://github.com/Constellation-Labs/tessellation/pull/809/files#r1453604537

If we do not check it and just use `IpAddress.fromString` empty value then we end up with `DecoderError` for empty cell which is not what we want according to what @cngo-github said 🤔

Reply:
We do not need this check. Re-worked to:

```scala
  implicit val maybeIpAddressCellDecoder: CellDecoder[Option[IpAddress]] =
    CellDecoder.stringDecoder.emap { value =>
      IpAddress
        .fromString(value)
        .asRight
    }
```